src/orders/components/OrderReturnPage/components/TransactionSubmitCard/TransactionSubmitCard.tsx

```diff
@@ -0,0 +1,168 @@
+import { DashboardCard } from "@dashboard/components/Card";
```

Comment:
brak down this component into smaller units, accordingly to their contexts eg:
- managing grand refund itself
- managing refund itself (seems like you already have it) 
- managing shipping costs